src/js/components/app.coffee

```diff
@@ -112,11 +128,24 @@ App = React.createClass
       if @state.sequenceId
         resources = "sequences"
         id = @state.sequenceId
-      url = @urlHelper.tocUrl(@state.laraBaseUrl, resources, id)
-      $.ajax
-        url: url
-        dataType: "jsonp"
+      tocUrl = @urlHelper.tocUrl(@state.laraBaseUrl, resources, id)
+      @apiCall
+        url: tocUrl
         success: setSequence
+        dataType: "jsonp"
+        errorContext: 'loading table of contents'
+
+  reportError: (url, errorContext, errors) ->
+    error =
+      url: url
+      errorContext: errorContext
+      status: errors?[0]?.status
+      errorString:  errors?[1]
+
+    console?.log?(error)
```

Comment:
Did you mean to leave the console log in here?  Should you use console?.error?(error) instead?